itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenarios.java

```diff
@@ -4294,6 +4295,9 @@ public void testMoveOptimizationIncremental() throws IOException {
 
   @Test
   public void testDatabaseInJobName() throws Throwable {
+    // Clean up configurations
+    driver.getConf().set(JobContext.JOB_NAME, "");
```

Comment:
can this clean up be done at tear down?

Reply:
came from HIVE-25000, Removed itself from my code